### PR TITLE
Extend coverage for ApiResource

### PR DESCRIPTION
**Pull request by raulsteurer**

### CODE REVIEW

File: src/main/java/com/stripe/BranchCoverageUtil.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
 import java.io.File;
 
 public class BranchCoverageUtil {
+  private final static int BRANCHES_COUNT = 10;
```
Comment (KacperMachaj1) [inline src/main/java/com/stripe/BranchCoverageUtil.java:16]: Smart design choice. I did not think about that idea. Good job!

File: src/main/java/com/stripe/net/ApiResource.java [hunk 2]
```diff
@@ -85,17 +86,22 @@ public static String urlEncode(String str) {
     // Preserve original behavior that passing null for an object id will lead
     // to us actually making a request to /v1/foo/null
     if (str == null) {
+      BranchCoverageUtil.insertXAtIndex(4);
```
Comment (KacperMachaj1) [inline src/main/java/com/stripe/net/ApiResource.java:89]: Good use of that function! Exactly as expected :)

**Review by KacperMachaj1 — APPROVED** [review]
Good Job mr Collaborator san. Pleassure working with you mr collaborator san.

File: src/main/java/com/stripe/BranchCoverageUtil.java (hunk 1 above)
Comment (raulsteurer) [reply src/main/java/com/stripe/BranchCoverageUtil.java:16]: That's OK! Everyone has to start from somewhere!

File: src/main/java/com/stripe/net/ApiResource.java (hunk 2 above)
Comment (raulsteurer) [reply src/main/java/com/stripe/net/ApiResource.java:89]: Thank you

**Pull request merged**